src/main/java/com/google/api/codegen/transformer/GeneratorVersion.java

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer;
+
+import java.io.IOException;
+import java.util.Properties;
+
+public class GeneratorVersion {
```

Comment:
This isn't a transformer (it takes no input!), so I think it should go up one directory in the general area. Since this class doesn't hold data and just provides it, I think it should probably follow the naming convention of "doer" or "X doer" as opposed to "X".